code/modules/overmap/sectors.dm

```diff
@@ -27,9 +29,25 @@
 	for(var/zlevel in map_z)
 		map_sectors["[zlevel]"] = src
 
+	// Spawning location of area is randomized or default values, but can be changed to the Eris Coordinates in the code below.
+	// This provides a random starting location for Eris.
 	start_x = start_x || rand(OVERMAP_EDGE, GLOB.maps_data.overmap_size - OVERMAP_EDGE)
 	start_y = start_y || rand(OVERMAP_EDGE, GLOB.maps_data.overmap_size - OVERMAP_EDGE)
 
+	if (!eris_start_set)
+		// Find the eris object
+		for (var/obj/effect/overmap/ship/eris/O in world)
+			if (istype(O, /obj/effect/overmap/ship/eris))
+				if(O.name == "CEV Eris")
```

Comment:
The whole way the ship gets found is kinda strange, but hopefuly random start will always stay on so it shouldn't be a big problem.

Reply:
I hope our code has this eris ship being already in a global var/list

Reply:
I can't find it in the global vars, so I made a new var list for ships and put it in there.
I added configuration file support. 
Now that the eris no longer needs to be 'found' that part is gone.